app/(dashboard)/dashboard/statistics/[userId]/[eventId]/page.tsx

```diff
@@ -21,11 +34,35 @@ export default function EventStatistics({ params }: { params: PageProps }) {
   return (
     <div className="p-4 ">
       <h1 className="pb-2 text-lg font-semibold text-accent">Attendee List</h1>
-      {attendees ? (
-        // <p>rendered</p>
 
-        // attendees
-        <AttendeeTable attendees={attendees} />
+      {attendees.length > 0 ? (
+        <>
+          {loading && (
+            <div className="grid grid-cols-2 gap-x-2 py-4">
+              <ChartContainer
+                heading="Sign Ups Per Day"
+                subheading="Total Sign Ups"
+                colSpan={1}
+                attendants={attendeesWithSubmittedDay}
+                label="Sign Ups"
+                param={'submittedDay'}
+                chartType="line"
+                type="length"
+              />
+              <ChartContainer
+                heading="Age"
+                subheading="Age"
+                colSpan={1}
+                attendants={attendees}
+                label="Count"
+                param={'age'}
+                chartType="bar"
+                type="select"
+              />
+            </div>
+          )}
+          <AttendeeTable attendees={attendees} />
+        </>
       ) : (
         <p>Loading attendees...</p>
```

Comment:
maybe slight change to "No attendees..." instead cuz i kept waiting for it to load LOL